File: retroravelry/src/main/java/com/caseykulm/retroravelry/models/Paginator.java
package com.caseykulm.retroravelry.models;

import com.google.auto.value.AutoValue;

@AutoValue public abstract class Paginator {

  public abstract int last_page();
  public abstract int page();
  public abstract int page_count();
  public abstract int page_size();
  public abstract int results();

}

File: retroravelry/src/main/java/com/caseykulm/retroravelry/models/Paginator.kt
package com.caseykulm.retroravelry.models

import com.squareup.moshi.Json

data class Paginator (
        @Json(name="last_page") val lastPage: Int,
        val page: Int,
        @Json(name="page_count") val pageCount: Int,
        @Json(name="page_size") val pageSize: Int,
        val results: Int
)
